pos_session_report/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Pos Session Report Detailed
+==============
+
+This module extends the functionality of Pos Session Sumary
+and to allow you to show detailed informations about pos session
```

Comment:
@Hendrixcosta, indeed, a print screen of a part of the generated pdf could be great to explain clearly what this module does.